CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 This file is used to list changes made in each version of auditd.
 
+## 2.3.4 (2018-11-8)
+
+- Added CIS-rules kitchen suite
```

Comment:
I would stick with one tense throughout - e.g add & update rather than added & updated - I think the former is preferred.